src/adapters/category.js

```diff
@@ -37,7 +37,12 @@ export default class Category {
     const prefixLength = Math.min(4, this.label.length);
     // Match label prefix (eg: "supe" for "Supermarché", but not "supel")
     if (!matched && term.length >= prefixLength) {
-      matched = ExtentedString.compareIgnoreCase(term, this.label.slice(0, term.length)) >= 0;
+      if (this.label.length < term.length) {
+        // check if first word in term is label
+        matched = ExtentedString.compareIgnoreCase(term, `${this.label} `) === 0;
```

Comment:
Not related to the feature, but there is a typo in the class name, should be `ExtendedString`.

Reply:
fixed